documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst

```diff
@@ -892,7 +787,9 @@ difference. The binning arguments of ``cut`` no longer refer to
    - :math:`\theta` runs between :math:`[0, 180]`
    - :math:`\phi` runs between :math:`[-180, 180]`
 
-   Attempting to specify binning outside of these ranges will fail.
+   Attempting to specify binning outside of these ranges will fail. Changing 
+   ``sphere_proj`` ``type`` property from ``add`` to ``arr`` modifies angular ranges
+   into radians.  
```

Comment:
what does this mean? Does it mean you have to input these angles in deg or rad, or that it is stored internally in deg or rad, or that you get output values in deg or rad, or what?

Reply:
hopefully modified the sentence to make it clearer